modules/acs-central-services-overview.adoc

```diff
@@ -40,17 +40,17 @@ You can use the same Central instance to secure multiple {ocp} or Kubernetes clu
 +
 [NOTE]
 ====
-The {product-title-short} Scanner V4 uses the OSV database available at link:https://osv.dev/[OSV.dev] under link:https://github.com/google/osv.dev/blob/master/LICENSE[this license].
+The {product-title-short} Scanner V4 also uses the OSV database available at link:https://osv.dev/[OSV.dev] under link:https://github.com/google/osv.dev/blob/master/LICENSE[this license].
 ====
-* link:https://nvd.nist.gov/vuln/search[NVD]: This used for various purposes such as filling in information gaps when vendors do not provide information. For example, Alpine does not provide a description, CVSS score, severity, or published date.
+* link:https://nvd.nist.gov/vuln/search[NVD]: This is used for various purposes such as filling in information gaps when vendors do not provide information. For example, Alpine does not provide a description, CVSS score, severity, or published date.
 +
 [NOTE]
 ====
 This product uses the NVD API but is not endorsed or certified by the NVD.
 ====
 * link:https://github.com/stackrox/stackrox/blob/master/scanner/updater/manual/vulns.go[StackRox]: The upstream StackRox project maintains a set of vulnerabilities that might not be discovered due to data formatting from other sources or absence of data.
 
-The Scanner V4 Indexer uses the following sources:
+The Scanner V4 Indexer uses the following files to index Red Hat containers:
```

Comment:
I'm not actually sure if this is a hard requirement: https://redhat-documentation.github.io/supplementary-style-guide/#non-breaking-spaces

```suggestion
The Scanner V4 Indexer uses the following files to index Red{nbsp}Hat containers:
```

Reply:
I always forget this! Thanks :)